jobserver/views/job_requests.py

```diff
@@ -283,12 +283,23 @@ def get_project_yaml(self, job_request):
 
         # We know files that are 2.43MB take too long to render with pygments
         # in production and crash browser tabs locally, so we're limiting the
-        # size.  The current 1MB is arbitrary until we can get some telemetry
+        # size.  The current ~10KB is arbitrary until we can get some telemetry
         # from production.
-        is_too_large = len(job_request.project_definition) > 10_000  # ~1MB
+        # Note: ~10KB assuming Latin-1 encoding, UCS-2 or UCS-4 would be ~2/4
+        # times as big. In any case, we're using length as only an approximate
+        # measure of compute required to render the definition.
```

Comment:
I had to read this comment a couple of times. Do you mean...

> We want to limit the project definition to 10kB. Assuming the Latin-1 encoding (1 byte/character), then 10kB corresponds to 10,000 characters. However, 10,000 characters correspond to 20kB in UCS-2 (2 bytes/character) or 40kB in UCS-4 (4 bytes/character).

Reply:
Thanks, Iain. This is much clearer. On reflection, it's more important to highlight the approximate nature of the length measure than how encoding impacts size. I've rewritten both comments to be more focused -- let me know your thoughts. And thanks for pushing me to rethink how these comments could better serve their purpose.